Main/ClientApp/src/Components/Dialogs/ShareCollectionDialog.tsx

```diff
@@ -35,7 +35,7 @@ export default function ShareCollectionDialog(
     return {
       collectionId: props.collectionId,
       userId: selectedUser!.id,
-      editRights: selectedUserRights === UserRights.EditRights,
+      editRights: !!UserRights.ViewRights,
```

Comment:
As previously discussed, this expression is equivalent to `false`, and I would recommend changing it to `false` to make it more clear.